developer_guides/topology2/topology2.rst

```diff
@@ -124,27 +124,27 @@ An example of a simple class definition with 2 attributes and qualifiers is as f
 		}
 	}
 
-The "data" class definition belonging to the "base" class group, contains 2 attributes namely,
-name and bytes, both of type "string". By default, all attributes are give the type "integer" unless
-specified otherwise like above. Currently, topology 2.0 supports only "integer" and "string" types for
-attributes.
+The ``data`` class definition belonging to the ``base`` class group contains two attributes,
+name and bytes, both of type ``string``. By default, all attributes have the ``integer`` type, unless
+specified otherwise, like in the example above. Currently, topology 2.0 supports only ``integer`` and
+``string`` types for attributes.
 
 The attribute qualifiers are used to describe how to instantiate an object from the class definition
 and validate the attribute values.
 
-In the above definition, the "constructor" array tells the compiler how to build the object's name.
-A data object instantiated with the name "EQIIR-Coefficients" will be given the name,
-"data.EQIIR-Coefficients" i.e. the class name followed by '.' followed by the constructor attribute
+In the above definition, the ``constructor`` array tells the compiler how to build the object's name.
+A data object instantiated with the name ``EQIIR-Coefficients`` will be given the name
+``data.EQIIR-Coefficients``, that is the class name followed by '.' followed by the constructor attribute
 values separated by '.'.
 
-The "unique" qualifier indicates that two data objects instantiated within the same alsaconf node should
-have unique values for their "name" attribute. If two data objects are instantiated within the same alsaconf
-node with the same "name" attribute, there be no errors but the two object instances with be merged
-with the second instance overriding the attribute values in the first one. Therefore, it is the topology
-writer's responbility to ensure that two instances within the same parent node have different unique attribute
-values.
+The ``unique`` qualifier indicates that multiple data objects instantiated within the same alsaconf node should
```

Comment:
@ranj063,  I've replaced "two" with "multiple" in this paragraph. Please verify that I did not change what you meant.